src/data/events/eventsReducer.ts

```diff
@@ -60,11 +60,11 @@ export default function reducer(state = initialState, action: any) {
   }
 }
 
-export const eventsFetchAsync = (filter: any) => {
+export const eventsFetchAsync = (filter: any, user?: any) => {
   return (dispatch: Dispatch<any>) => {
     search(
       remoteRoutes.events,
-      filter,
+      {filter, user},
```

Comment:
The user object is already available on the server side, there is no need to send it here. `req.user`